shell.nix

```diff
@@ -20,6 +23,15 @@ let inherit (nixroot) stdenv pkgs lib
     python3 python37Packages
     wavpack;
 
+  cmakeBuildType = if buildType == "auto" then (
+                      if releaseMode then "RelWithDebInfo" else "Debug"
```

Comment:
I'm not sure if this automation is important.

Note that the option `releaseMode` is named a bit misleading. It means that the expression can be built using `nix-build` instead of `nix-shell`. (I initially proposed to name this option `buildable` instead of `releaseMode`.) Therefore that option was never meant to have something to do with performance.

If someone still wants to set `-DCMAKE_BUILD_TYPE=RelWithDebInfo`, then he can still supply it assuming we introduce a `cFlags` parameter to this expression.

Reply:
People who use this for personal builds are usually developers and like debug informations. If you want a different build type you can specify it as argument.
I was very annoyed having no debug symbols by default after a cleanup in a dev shell.

Reply:
Oh, does the default have no debug information?

Reply:
No. It defaults to a Release build i think.